Core/src/Geometry/TrackingGeometry.cpp

```diff
@@ -16,13 +16,15 @@
 Acts::TrackingGeometry::TrackingGeometry(
     const MutableTrackingVolumePtr& highestVolume,
     const IMaterialDecorator* materialDecorator,
-    const GeometryIdentifierHook& hook)
+    const GeometryIdentifierHook& hook, const Logger& logger)
     : m_world(highestVolume),
       m_beam(Surface::makeShared<PerigeeSurface>(Vector3::Zero())) {
+  std::unique_ptr<const Logger> mylogger = logger.clone("TrackingGeometry");
+
   // Close the geometry: assign geometryID and successively the material
   size_t volumeID = 0;
-  highestVolume->closeGeometry(materialDecorator, m_volumesById, volumeID,
-                               hook);
+  highestVolume->closeGeometry(materialDecorator, m_volumesById, volumeID, hook,
+                               *(mylogger.get()));
```

Comment:
Depending on if you changed `mylogger` or not, you might not need the `.get()`:
```suggestion
                               *(mylogger));
```

Reply:
Yes. My intention was to update the name. So there is a clone.